src/style.css

```diff
@@ -1 +1,108 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  /* margin: 0 auto; */
+  /* background-color: lightgrey; */
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  max-width: 1200px;
+  min-width: 1024px;
+  height: 60px;
```

Comment:
What about this requirement?

![image](https://user-images.githubusercontent.com/98764641/210992932-f7a0c8f9-34b5-4c33-8ef3-fc73f6ef4a2d.png)

Reply:
which property i should use ?